### ISSUE TITLE
Filter.js crashes upon non existing discord users/channels

### ISSUE THREAD

**Issue by jepke** [KartulUdus/PoracleJS#18]
## Current Behavior
https://pastebin.com/raw/pniBxzcX

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Environment name and version (e.g. npm 5.6.0, nodejs v7.4.0):
* Operating System

**Comment by KartulUdus** [comment]
Tried to reproduce with channels|users

- created channel
- registered channel and set tracking
- sent some test hooks
- deleted channel 
-sent another hook

```
info: Pidgey appeared and 2 humans cared
info: Alerted super-secret about Pidgey monster
warn: Tried to send message to ID 519205815023042560, but error ocurred
info: Alerted testforeachissue about Pidgey monster

```

- added user to discord server
- registered trackings
- sent test hooks
- left server

```
debug:  id=519213933559676949, pokemon_id=16, distance=9999999, min_iv=-1, max_iv=100, min_cp=0, max_cp=9000, min_level=0, max_level=40, atk=0, def=0, sta=0, min_weight=0, max_weight=9000000, form=0, name=kartultest, alerts_sent=1, enabled=1, map_enabled=1, area=[], latitude=0, longitude=0
info: Alerted kartultest about Pidgey monster
debug: Added one to humans.alerts_sent
debug: Added one to humans.alerts_sent
(node:4108) UnhandledPromiseRejectionWarning: DiscordAPIError: Cannot send messages to this user
    at item.request.gen.end (/Users/name/kartuludus/PoracleJS/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:79:15)
    at then (/Users/name/kartuludus/PoracleJS/node_modules/snekfetch/src/index.js:215:21)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
(node:4108) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
(node:4108) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

reproduction steps needed

**Comment by KartulUdus** [comment]
seems to be fixed in 9f00f089bd14b11093b21365e5db21b7ca2de3fd
